IBX-7076: Customizable dashboard described in User Documentation (#246)
ibexa/documentation-user: docs/getting_started/dashboard_block_reference.md

---
description: Pick from a list of dynamic blocks to create customized dashboard.
edition: experience
---

# Dashboard block reference

The following blocks are provided with a clean installation of [[= product_name =]]:

|Block|Description|
|-----|-----------|
|[Common content](#common-content-block)|Displays a list of content items created by all the organization that the user belongs to.|

> |[Ibexa News](#ibexa-news-block)|Displays a list of recent blog posts or articles published at `ibexa.co` blog.|

Reviewer: ```suggestion
|[Ibexa News](#ibexa-news-block)|Displays a list of recent blog posts or articles published at `ibexa.co` blog (product related).|
```

|[Products with lowest stock](#products-by-category-block)|Displays a table presenting products with the lowest stock.|
|[Quick actions](#quick-actions-block)|Displays most popular/used actions and shortcuts.|
|[Recent activity](#recent-activity-block)|Displays a list of recent activity.|
|[Recent orders](#recent-orders-block)|Displays a table presenting recent orders.|
|[Review queue](#review-queue-block)|Displays a list of Content items which user or User group can review.|
|[Top 10 clicked items](#top-10-clicked-items-block)|Displays a table presenting top 10 clicked items.|

!!! note

    Before you add a block that involves products, product types, or product categories, make
    sure your that your [user Role](../permission_management/permissions_and_users.md) has
    the `Product/View` and `Product type/View` permission.

## Common content block

Displays a list of content items created by all the organization that the user belongs to.
It shows the following tabs: Content, Scheduled, Media.

On the **Properties** tab, set values in the following fields:

- **Name** - Enter a name for the block.

> On the **Design** tab, in the **View** field, select the layout to be used to present a list of content and submit your changes.

Author: Let's discuss "block_reference.md" first.

## Ibexa News block

Presents a list of recent blog posts or articles published at `ibexa.co` blog.

> It includes title, image, timestamp, and link to article details.

Reviewer: ```suggestion
It includes title, image, publication date, and link to article details.
```

## Products with lowest stock block

Displays a table of products with the lowest stock.
Products are sorted based on Stock value (sorted from lowest to highest stock).

Table contains following columns: Name, Image, Code, Category, Type, Variant, Stock.

On the **Properties** tab, set values in the following fields:

- **Name** - Enter a name for the block.
- **Number of products to be displayed** - Set the number of products to be displayed. Default value = 10.
- **Stock** - Enter the stock number for a product. Default value = 10.

On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.

## Quick actions block

Displays most popular/used actions and shortcuts, for example, **Create content**.

On the **Properties** tab, set values in the following fields:

- **Name** - Enter a name for the block.
- **Actions** - Select actions to be displayed as shortucts: Create content, Create form, Create product, Create catalog, Create company.

On the **Design** tab, in the **View** field, select the layout to be used to present a list of quick actions and submit your changes.

## Recent activity block

> Displays a list of recent activity of Users.

Reviewer 2: ```suggestion
Displays a list of recent activity of users.
```

It also includes the link to view All activities available in Admin tab.

Recent activity block contains the following data: action time, User reference (avatar, first and last name) with link to the user profile (if available),
Activity type with the context.

> Available filters are: Users, Activity type, Activity area (default value = All), Number of activities (default value = 5, maximum value = 10).

Reviewer 2: As far as I know, the limit now is 6 but please verify it with the Design team.

- **Limit** - Set the maximum number of orders to be displayed. Default value = 10.

On the **Design** tab, in the **View** field, select the layout to be used to present a list of orders and submit your changes.

## Review queue block

Displays a list of Content items which user or User group can review.

On the **Properties** tab, set values in the following fields:

- **Name** - Enter a name for the block.

On the **Design** tab, in the **View** field, select the layout to be used to present a list of content items for review and submit your changes.

## Top 10 clicked items block

Displays a table presenting top 10 clicked items.

Table contains following columns: Item clicked (including item name), Item type, Recommended (presenting number of recommendations of selected item),
Clicked (presenting total number of clicks on selected item).

Available filters are: Item types, Customer ID, Number of displayed items (default value = 10), Time period.

On the **Properties** tab, set values in the following fields:

- **Name** - Enter a name for the block.

> - **Customer Id** - Select customer Id whose top 10 clicks are displayed.

Author: Reported
